.github/configuration/upgrade-test-configuration.yml

```diff
@@ -38,6 +38,16 @@ upgrade-version: [[
     upgrade-type: minor
   }
 ],
+[
```

Comment:
I think this seems unnecessary to add an additional workflow, instead let's update the above entry from 14.8 -> latest to 14.9->latest as 14.3/14.5 -> latest already covers intermediate paths.